implementation/core/src/main/java/io/smallrye/faulttolerance/core/circuit/breaker/CircuitBreakerLogger.java

```diff
@@ -0,0 +1,56 @@
+package io.smallrye.faulttolerance.core.circuit.breaker;
+
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRFTL", length = 5)
+public interface CircuitBreakerLogger {
+    CircuitBreakerLogger log = Logger.getMessageLogger(CircuitBreakerLogger.class,
```

Comment:
Given this is static, it should be `LOG`.

Reply:
It probably should, but it seems SmallRye project use lower-case in this case. See https://github.com/smallrye/smallrye/wiki/I18N-Messages-and-Logging or https://github.com/smallrye/smallrye-common/search?q=%22%40MessageBundle%22

I personally don't mind either way.

Reply:
I also did this. All singleton logger instances are now called `LOG`, and are statically imported, leading to much nicer code.